scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -297,15 +297,19 @@ function Deploy-IoTEdge {
         [HashTable] $InvokeWebRequestParameters,
 
         # Restart if needed without prompting.
-        [Switch] $RestartIfNeeded
+        [Switch] $RestartIfNeeded,
+
+        # Skip processor architecture check.
+        [Switch] $SkipArchCheck
```

Comment:
Why do we allow to skip for deploy but not initialize?

Reply:
I figured if `Deploy-` has succeeded, there's no reason to require the user to specify `-SkipArchCheck` again for `Initialize-`. So `Initialize-` always sets the switch.

Reply:
Ah, though I just realized `Update-` should also always set it.

Reply:
Sounds good.